docs/how-to-guides/training-machine-learning-models/training-models.md

```diff
@@ -28,8 +28,17 @@ an example dataset containing three distinct classes (green, yellow, red), which
 
 Detailed instructions for training the traffic light classifier model can be found **[here](https://github.com/autowarefoundation/autoware.universe/blob/main/perception/traffic_light_classifier/README.md)**.
 
-<!--
+## Training CenterPoint 3D object detection model
 
-Training traffic light detection model and lidar CenterPoint model will be added there.
+The CenterPoint 3D object detection model within the Autoware has been trained using the **[open-mmlab/mmdetection3d](https://github.com/open-mmlab/mmdetection3d)** repository. CenterPoint
+implementation of mmdetection3d uses 10 input features for the PointPillars voxel encoder. However, Autoware employs 9 pillar features for CenterPoint, in accordance with the original research paper. Therefore, we have
```

Comment:
> 10 input features

This expression is ambiguous. What is 10 input features?

Reply:
@Shin-kyoto -san, Thank you, I changed the paragraph a little bit to make the documentation more clear.

Reply:
@kaancolak 
Thanks! Can you move the reason why we have to make the training repository for centepoint to https://github.com/autowarefoundation/mmdetection3d ?

Reply:
@Shin-kyoto  -san,  I refactored the paragraph, moved the first part to the ReadMe file of the AutowareCenterPoint project, and linked with it. 

https://github.com/autowarefoundation/mmdetection3d/pull/1/files#diff-c1d68ca9b975ba79c77f68690b85862e68eecfe664a6a711d02121bf4093c0b9R1